src/ComponentInstaller.php

```diff
@@ -178,6 +179,9 @@ public function onPostPackageInstall(PackageEvent $event)
             return;
         }
 
+        $this->includeModuleClasses($package);
```

Comment:
This is here, because later (in loop for each module) we don't have access to `$package`.